charmhelpers/core/hookenv.py

```diff
@@ -378,7 +384,10 @@ def config(scope=None):
         if scope is not None:
             return _cache_config.get(scope)
         return _cache_config
-    except ValueError:
+    except (exc_json, UnicodeDecodeError) as e:
+        log('Unable to parse output from config-get: config_cmd_line="{}" '
+            'message="{}"'
+            ''.format(config_cmd_line, str(e)), level=ERROR)
```

Comment:
You don't need the `''` before the `.format(...)`

Reply:
Thx!